src/openrct2/ride/Ride.cpp

```diff
@@ -98,8 +98,8 @@ static constexpr const int32_t RideInspectionInterval[] = {
 // Ride storage for all the rides in the park, rides with RideId::Null are considered free.
 static std::array<Ride, OpenRCT2::Limits::MaxRidesInPark> _rides{};
 
-// This is not the real size of rides in use, rather the highest slot used + 1.
-static size_t _maxRideSize = 0;
+// This is the highest used index + 1 of the _rides array.
+static size_t _maxHighestUsedSlot = 0;
```

Comment:
'Max' suggests that it's an absolute maximum possible, but it's pretty much the end of use the used range if I understand this correctly. Perhaps a name like `_endOfUsedRange` or something similar better explains what this is exactly.

Reply:
Changed